### PR TITLE
Document the display_override manifest member

### LINKED ISSUE #10539: Content suggestion: Window Controls Overlay for PWAs
## What is the new suggestion?
I'm proposing to add some documentation content for the Window Controls Overlay feature of PWAs.

This feature is implemented in Chromium, and while it's been on origin trial up until now, is going to be globally available in Chrome and Edge soon.

It allows PWAs on desktop to make use of the top window area normally reserved for the title bar.
The feature is documented in these places right now:

* https://web.dev/window-controls-overlay/
* https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay

And is spec'd as a WICG explainer: https://github.com/WICG/window-controls-overlay/blob/main/explainer.md and wicg spec draft: https://wicg.github.io/window-controls-overlay/

## Why is it important or useful?

The feature is implemented, and changes how PWAs look on desktop quite a lot. It gives web authors opportunities to really make their apps "native"-looking. It's important to document the various APIs that it depends on.

The main changes that would be needed are:

- [x] Adding the `display_overrides` member to https://developer.mozilla.org/en-US/docs/Web/Manifest
- [x] Documenting the `window-controls-overlay` value on the `display_overrides` page
- [x] Adding the `titlebar-area-*` environment variables to https://developer.mozilla.org/en-US/docs/Web/CSS/env()
- [x] Documenting the `navigator.windowControlsOverlay` API

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary
I am adding some documentation for the `display_override` PWA manifest member. This member was adding in Chromium in 89. I am also adding compat data for it in this PR: https://github.com/mdn/browser-compat-data/pull/13565 

#### Motivation
This helps make the manifest documentation more complete. `display_override` has been around for some time, and is starting to be used in an even more meaningful way with the Window Controls Overlay feature.

#### Supporting details
* web.dev article: https://web.dev/display-override/
* WICG spec: https://wicg.github.io/manifest-incubations/index.html#display_override-member

#### Related issues
Part of #10539.

#### Metadata

- [x] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by captainbrosset** [comment]
>     * No BCD data for query: html.manifest.display_override

This is taken care of in mdn/browser-compat-data#13565 and we'll probably need to wait for it to land first, and then make its way to the mdn/content repo.

**Comment by captainbrosset** [comment]
BCD data just got merged, so we now need to wait for MDN to get the update.

File: files/en-us/web/manifest/display_override/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  </tbody>
+</table>
+
+The [`display`](/en-US/docs/Web/Manifest/display) member is used to determine the developer's preferred display mode for a website. It follows a process where the browser falls back to the next display mode if the requested one is not supported. In some advanced use cases, this fallback process might not be enough.
+
+The `display_override` member solves this by letting the developer provide a sequence of display modes that the browser will consider before using the `display` member. Its value is an array of display modes that are considered in-order, and the first supported display mode is applied.
+
+## Values
+
+Display override objects are display mode strings, the possible values are:
```
Comment (sideshowbarker) [inline files/en-us/web/manifest/display_override/index.md:32]:
```suggestion
Display override objects are display-mode strings, the possible values are:
```

File: files/en-us/web/manifest/display_override/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        depending on the browser and platform. This is the default.
+      </td>
+      <td>(None)</td>
+    </tr>
+  </tbody>
+</table>
+
+## Example
+
+In the example below, the browser will consider the following display mode fallback chain in this order: `fullscreen` → `minimal-ui` → `standalone`.
```
Comment (sideshowbarker) [inline files/en-us/web/manifest/display_override/index.md:84]:
```suggestion
In the example below, the browser will consider the following display-mode fallback chain in this order: `fullscreen` → `minimal-ui` → `standalone`.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by captainbrosset** [comment]
Thank you @sideshowbarker for making those changes and merging!
Just one question: the compat and spec data that this documentation relies on is part of [BCD 4.1.1](https://github.com/mdn/browser-compat-data/releases/tag/v4.1.1), but the version of yari MDN currently depends on only uses 4.1.0.

**Comment by sideshowbarker** [comment]
We’d first need a new Yari release that includes BCD 4.1.1, and that release would need to be published by the @mdn/core-yari-dev team

**Comment by schalkneethling** [comment]
> We’d first need a new Yari release that includes BCD 4.1.1, and that release would need to be published by the @mdn/core-yari-dev team

Done. https://github.com/mdn/yari/pull/5045

**Comment by sideshowbarker** [comment]
> > We’d first need a new Yari release that includes BCD 4.1.1, and that release would need to be published by the @mdn/core-yari-dev team
> 
> Done. [mdn/yari#5045](https://github.com/mdn/yari/pull/5045)

Thanks — updated mdn/content in https://github.com/mdn/content/pull/11193 / https://github.com/mdn/content/commit/b528bf573a7a95b9d909258ed9130867aedab2ca

**Pull request merged**